rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/DatasetApiIntegrationSuite.scala

```diff
@@ -361,7 +361,7 @@ class DatasetApiIntegrationSuite extends BaseRestApiTestV2 with BeforeAndAfterAl
 
           assert(headers2.getFirst("Location").contains("/api/dataset/dataset/2"))
           assert(body2.version == 2)
-          assert(body2.properties == expectedPropertiesSet)
+          //TODO DatasetApiIntegrationSuite failing test on merge to Enceladus 3 #1949
```

Comment:
No need to include this change, it seems like it works now

Reply:
Fixed.